docker-compose.ci.yml

```diff
@@ -1,7 +1,56 @@
 services:
+  serve:
+    build:
+      cache_from:
+        - type=gha
+      cache_to:
+        - type=gha,mode=max
```

Comment:
I think we need to swap `type=gha` in these for cache `type=registry, ref=orionrobots/orionrobots-site:cache`. Can we try that?